src/SeleniumLibrary/keywords/element.py

```diff
@@ -325,7 +326,7 @@ def element_should_not_be_visible(
     def element_text_should_be(
         self,
         locator: Union[WebElement, str],
-        expected: str,
+        expected: Union[None, str],
```

Comment:
Doesn't this now mean that you cannot test that element text is string `none`?

This is to some extend related to the open issue about changing `Union` conversion semantics (robotframework/robotframework#3897). If we'd change the logic so that conversion isn't done if an argument has one of the accepted types, using `Union[str, None]` here would allow using `${NONE}` (i.e. real Python `None`) but strings were never converted to `None`.

Reply:
Yes, string `None` can not be tested. But that how it did work when SL did have the custom `is_noney` in place. Also the difficult part is that Selenium will return Python `None` when something is not found. 

Also there is some logic in SL that relies that string None is converted to Python `None ` in some other keywords.

Reply:
Not being able to test string `None` sounds like an annoying limitation to me. Let's discuss on Slack how we can make that work. This will help deciding about robotframework/robotframework#3897 as well.

Reply:
I agree lest discuss in Slack, it is easier. But this is mostly just returning functionality of SeleniumLibrary did work before RF 4 and SL 5.0.